packages/@react-stately/layout/src/ListLayout.ts

```diff
@@ -377,14 +377,6 @@ export class ListLayout<T> extends Layout<Node<T>> implements KeyboardDelegate,
     }
   }
 
-  getKeyLeftOf(key: Key): Key {
-    return key;
-  }
-
-  getKeyRightOf(key: Key): Key {
-    return key;
-  }
-
```

Comment:
Unnecessary now that useGridListItem handles left/right arrow keys presses in ListView rather than using useGridCell. Normal lists that use ListLayout won't have elements within their rows so they won't even use useGridCell

Would've written a test but didn't get accurate `selectionStart` returned by onKeyUp when triggering arrowleft/right. A updated version of user test library should fix this theoretically but that will require a greater refactor of our tests so I'll handle that later